### PR TITLE
Hommeage au nepal

### PR DESCRIPTION
**Pull request by uchadh**

### CODE REVIEW

**Review by adrinz — APPROVED** [review]
Approved

File: _posts/2020-06-20-Hommage au Népal.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Hommage au Népal"
+date: 2016-03-20
+description: First review
+image: /assets/images/HommageAuNepal.jpg
+author: Dasol
+movie:
```
Comment (jungeebah) [inline _posts/2020-06-20-Hommage au Népal.md:8]: why was the santadard not followed here what was the reason

**Review by jungeebah — COMMENTED** [review]
Why was it approved without looking at the environment what is the use of all this stuff if you blindly trust your friend

**Comment by uchadh** [comment]
K chai bhaena bijen yesmaa??

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**